colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -1166,6 +1167,12 @@ def configure(
                     **self.zero_config,
                     **self.amp_config,
                 )
+                # Setup optimizers that require global states
+                if isinstance(optimizer.optim, DistributedOptim):
+                    tp_group = self.__dict__.get("tp_group", None)
+                    dp_group = self.__dict__.get("dp_group", None)
+                    shard_to_param = optimizer._param_store.master_to_working_param
+                    optimizer.optim.setup_distributed(tp_group, dp_group, shard_to_param)
```

Comment:
This should be called even if zero_stage == 0

Reply:
fixed